### ISSUE TITLE
react-native link stuck

### ISSUE THREAD

**Issue by mauricioscotton** [react-native-community/cli#89]
- [YES] Review the documentation: https://facebook.github.io/react-native
- [YES] Search for existing issues: https://github.com/facebook/react-native/issues && https://github.com/react-native-community/react-native-cli/issues
- [YES] Use the latest React Native release: https://github.com/facebook/react-native/releases
- [YES] Googled A-LOT for this bug, nothing found.
- [YES] Posted in the wrong place? 

## Environment 
![image](https://user-images.githubusercontent.com/34411735/51138703-17570d80-1842-11e9-85a6-d9631bbda147.png)
![image](https://user-images.githubusercontent.com/34411735/51138724-20e07580-1842-11e9-94bc-30075f48294c.png)

## Description
I've just started a new project and started developing the base classes...
So, I installed react-navigation (using yarn and also npm), and got a error regarding a State, so I've figured out that I was missing a "react-native link".
When I ran this command, It got stuck...
Nothing was shown on terminal.
So, I've added the 'rnpm' assets on package.json to try out if I get something on console, and then after linking the assets it just hanged after:
"rnpm-install info Assets have been successfully linked to your project"

Then, I though I was kinda crazy, and googled tons of times, but no success.
After that, I've created this project "ABACATE" fresh to check if it was something with the new project that I had created before, but unfortunately I THINK that it is a bug. (Could it be a node version problem?) :D
![image](https://user-images.githubusercontent.com/34411735/51139050-1a063280-1843-11e9-84db-3d6622044900.png)
![image](https://user-images.githubusercontent.com/34411735/51139064-25f1f480-1843-11e9-8dae-36f743b6aaef.png)

Full Log here: https://pastebin.com/jm0A4GVC

## Reproducible Demo
react-native init YourProjectThatIsGoingToHangOnLink
react-native link

##Package.json:
{
  "name": "ABACATE",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest"
  },
  "dependencies": {
    "react": "16.6.3",
    "react-native": "0.57.8"
  },
  "devDependencies": {
    "babel-jest": "23.6.0",
    "jest": "23.6.0",
    "metro-react-native-babel-preset": "0.51.1",
    "react-test-renderer": "16.6.3"
  },
  "jest": {
    "preset": "react-native"
  }
}

##REQUEST:
Can you guys please add a more detailed log/verbose on the command "react-native link"?
It will make easier for all of us to trace errors... 

Thanks a million!

**Comment by mauricioscotton** [comment]
Guys, my guess was right.
I've installed NVM4W (Node Version Manager 4 Windows),
Then, I've installed Node v10.15.0 and ran: 
_nvm use v10.15.0_
_react-native link_
And the expected behaviour:
![image](https://user-images.githubusercontent.com/34411735/51171706-7ef47380-18a9-11e9-9336-c5da21199cf1.png)
![image](https://user-images.githubusercontent.com/34411735/51171731-90d61680-18a9-11e9-9419-7de0f706d406.png)

So now, the bug is reduced to new version of CLI with old Node v8.11.3..

Are you guys able to provide more verbose/log to this command?
It will be helpfull for all of us!

Thanks in advance
Cheers

**Comment by mauricioscotton** [comment]
Guys, my guess was right.
I've installed NVM4W (Node Version Manager 4 Windows),
Then, I've installed Node v10.15.0 and ran: 
_nvm use v10.15.0_
_react-native link_
And the expected behaviour:
![image](https://user-images.githubusercontent.com/34411735/51172101-66d12400-18aa-11e9-9078-6a7f3076ab30.png)

So now, the bug is reduced to new version of CLI with old Node v8.11.3..

Are you guys able to provide more verbose/log to this command?
It will be helpfull for all of us!

Thanks in advance
Cheers
